openstack/cronus/templates/rhea/_config.yaml.tpl

```diff
@@ -19,6 +19,23 @@ rhea:
     endpointType: {{ .Values.config.endpointType }}
 {{- end }}
   auth:
+    keystone:
```

Comment:
SHould this "keystone" be 1 more line down? I.E. under the `{{- if .Values.config.keystone }}`? Or is a value that is always expected by Rhea?

Reply:
It is always expected.